extensions/src/teachableMachine/index.ts

```diff
@@ -149,8 +147,11 @@ export default class teachableMachine extends extension({
     const { model, modelType } = this.predictionState[modelUrl];
     switch (modelType) {
       case this.ModelType.IMAGE:
-        const imageBitmap = await createImageBitmap(frame);
-        return await model.predict(imageBitmap);
+        if (frame) {
```

Comment:
FYI: I often like to avoid nesting (sometime's to a fault), so I might implement this as:
```ts
if (!frame) return null;
const imageBitmap = await createImageBitmap(frame);
return await model.predict(imageBitmap);
```

Reply:
Pushed!